api_fhir_r4/tests/test/test_claim.json

```diff
@@ -1,123 +1,205 @@
 {
-  "resourceType": "Claim",
-  "billablePeriod": {
-    "end": "2019-06-12T00:00:00",
-    "start": "2019-06-01T00:00:00"
-  },
-  "created": "2019-06-12T00:00:00",
-  "diagnosis": [
-    {
-      "diagnosisCodeableConcept": {
-        "coding": [
-          {
-            "code": "ICD_CD"
-          }
-        ]
-      },
-      "sequence": 1,
-      "type": [
+    "resourceType": "Claim",
+    "identifier": [
         {
-          "text": "icd_0"
+            "type": {
+                "coding": [
+                    {
+                        "system": "https://openimis.github.io/openimis_fhir_r4_ig/CodeSystem/openimis-identifiers",
+                        "code": "MR"
```

Comment:
We don't have this code anymore. Please update.

Reply:
ok